### PR TITLE
BC-2882 apply data-testid on the web elements related to Teams

### PR DESCRIPTION
**Pull request by atlldwp**
# Description
Add data-testids on teams relevant web elements

## Links to Tickets or other pull requests

https://ticketsystem.dbildungscloud.de/browse/BC-2882
https://github.com/hpi-schul-cloud/nuxt-client/pull/2344
## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: views/administration/dashboard.hbs [hunk 1]
```diff
@@ -72,7 +72,7 @@
 									<div class="card h-100 teacher-option locationlink" data-loclink="/administration/school">
 										<div class="card-block homework">
 											<div class="card-text">
-												<h2 class="h4"><i class="fa fa-building"></i> {{$t "administration.dashboard.headline.manageSchool" }}</h2>
```
Comment (MartinSchuhmacher) [inline views/administration/dashboard.hbs:75]: better put testid on the parent div object

**Review by MartinSchuhmacher — COMMENTED** [review]
Very good work so far @atlldwp  👍 
just one remark as the whole div is clickable and can be used as a link, will work on the rest by myself

File: views/lib/forms/form-create-videoconference.hbs [hunk 2]
```diff
@@ -6,17 +6,17 @@
 	<div class="create-videoconference">
 		<div class="checkbox create-videoconference-option">
 			<p id="startMuted" class="create-videoconference-option-text">{{$t "lib.form_create_videoConference.text.muteParticipants" }}</p>
-            <input aria-labelledby="startMuted" name="startMuted" type="checkbox" data-toggle="toggle" data-on="{{$t "global.button.yes" }}" data-off="{{$t "global.button.no" }}">
+            <input aria-labelledby="startMuted" name="startMuted" type="checkbox" data-toggle="toggle" data-on="{{$t "global.button.yes" }}" data-off="{{$t "global.button.no" }}" data-testid="toggle_mute_participants" checked>
```
Comment (odalys-dataport) [inline views/lib/forms/form-create-videoconference.hbs:9]: Is it correct to add "checked" here?

**Review by odalys-dataport — APPROVED** [review]

File: views/lib/forms/form-create-videoconference.hbs (hunk 2 above)
Comment (MartinSchuhmacher) [reply views/lib/forms/form-create-videoconference.hbs]: you're right, will fix that as discussed

**Pull request merged**